### ISSUE TITLE
[Bug]: <numpy.unique() doesn't work as expected>

### ISSUE THREAD

**Issue by ranahgawad** [unifyai/ivy#13921]
### Bug Explanation

The `numpy.unique()` function doesn't work as expected when passed an axis argument. Always acts as if `axis = None` even when any number is passed.

![image](https://user-images.githubusercontent.com/64759197/230819376-557de340-0ff9-4b1f-9564-b2c911a73b2b.png)

### Steps to Reproduce Bug

Try running the code in the following screenshot, where `unique` is the function implemented in numpy's frontend, always acts as if `axis = None`

![image](https://user-images.githubusercontent.com/64759197/230819555-b38cac5e-5fda-4c16-9194-57216086ae93.png)
 

### Environment

Windows 10 using Codespaces on VSCode

### Ivy Version

v1.1.9

### Backend

- [X] NumPy
- [ ] TensorFlow
- [ ] PyTorch
- [ ] JAX
